File: ExhamHorror/Assets/Scripts/FallDamage.js
#pragma strict

//IGD Valtter Taipale 2013

public var healthImput : PlayerHealth;

public var falling : boolean = false;

public var fallingDamage = 10f;

function Start () {

}

function OnTriggerEnter(){

	falling = true;


}

function Update () {

	if (falling == true) {
	
		healthImput.TakeDamage(fallingDamage);

	}

}

File: ExhamHorror/Assets/Scripts/FallDamage.cs
using UnityEngine;
using System.Collections;

// IGP Valtter Taipale 2013


public class FallDamage : MonoBehaviour {

	public PlayerHealth healthImput;

	public bool falling = false;
	
	public float fallingDamage = 10f;

	void Start () {
		
	}	
	
	void OnTriggerEnter (Collider other) {
		
		if(other.gameObject.tag == "Player") 
		{
			falling = true;
		}
	}
	
	void Update () {

		if (falling == true) 
		{
			healthImput.TakeDamage(fallingDamage);
		}

	}
}
